deploy/charts/hostpath-provisioner/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for hostpath-provisioner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+provisioner:
+  image:
+    repository: cdkbot/hostpath-provisioner
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "1.4.2"
+
+busybox:
+  image:
+    repository: busybox
+    tag: "1.28.4"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+storageClass:
+  name: "hostpath-storage"
+  isDefault: true
+  provisioner: "k8s.canonical.com/hostpath"
+  reclaimPolicy: "Delete"
+  pvDir: "/var/snap/k8s/common/default-storage"
+
+clusterRole:
+  name: "hostpath-provisioner"
+
+clusterRoleBinding:
+  name: "hostpath-provisioner"
```

Comment:
I would recommend that these are not config options, but are instead set to something specific like

```
{{ .Release.Name }}-sa
{{ .Release.Name }}-clusterrole
{{ .Release.Name }}-clusterrolebinding
```

etc

We would not want people that upgrade/manage the chart to leave behind or overwrite existing service accounts